components/Tabs/examples/supplemental/TabBarViewControllerExampleSupplemental.m

```diff
@@ -78,6 +84,17 @@ - (void)viewWillAppear:(BOOL)animated {
 - (void)viewDidLayoutSubviews {
   [super viewDidLayoutSubviews];
   _titleLabel.center = self.view.center;
+  UIEdgeInsets edgeInsets = UIEdgeInsetsZero;
```

Comment:
Nit/nameing: `safeAreaInsets`

Reply:
Addressed in a46864d6635c5903a535e6436aed7fb783f084b3.